### PR TITLE
Fix margins in course widget on API 15

### PR DESCRIPTION
**Pull request by eadm**
**YouTrack task**: [#APPS-1472](https://vyahhi.myjetbrains.com/youtrack/issue/APPS-1472)

**Description List**:
* Fixed margins in course widget on API 15

### CODE REVIEW

File: app/src/main/res/layout/new_course_item.xml [hunk 1]
```diff
@@ -59,6 +61,7 @@
         android:paddingEnd="0dp"
```
Comment (KirillMakarov) [inline app/src/main/res/layout/new_course_item.xml]: I think mistake is here. `right` and `end` should be equal. We should use `@dimen/extraPaddingRightCourseItem`

File: app/src/main/res/layout/new_course_item.xml [hunk 2]
```diff
@@ -59,6 +61,7 @@
...
         android:paddingRight="8dp"
         android:paddingStart="16dp"
+        android:paddingLeft="16dp"
```
Comment (KirillMakarov) [inline app/src/main/res/layout/new_course_item.xml]: use `@dimen/guideline_standard_padding`

File: app/src/main/res/layout/new_course_item.xml (hunk 1 above)
Comment (eadm) [reply app/src/main/res/layout/new_course_item.xml]: Yes I have noticed that but I thought maybe there is some sense in that values

Comment (KirillMakarov) [reply app/src/main/res/layout/new_course_item.xml]: just bug :(

**Comment by KirillMakarov** [comment]
Fixed in https://github.com/StepicOrg/stepik-android/pull/113. Please, review changes. I will close this PR for avoiding double work

**Pull request closed without merging**